LCM/dsc/engine/ConfigurationManager/LocalConfigManagerHelper.c

```diff
@@ -233,12 +233,14 @@ MI_Result InitHandler(
         // already initialized.
         return MI_RESULT_OK;
     }
-    else if (initState == RUNNING_INITIALIZATION)
+    else if (initState == RUNNING_INITIALIZATION)   
```

Comment:
Removed. 

---
In reply to: [107557147](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/235#discussion_r107557147) [](ancestors = 107557147)